test/Ocelot.IntegrationTests/AdministrationTests.cs

```diff
@@ -882,7 +882,7 @@ private void ThenTheStatusCodeShouldBe(HttpStatusCode expectedHttpStatusCode)
         private void ThenTheResultHaveMultiLineIndentedJson()
         {
             const string indent = "  ";
-            const int total = 46, skip = 1;
+            const int total = 47, skip = 1;
```

Comment:
I recommend to switch off the step!
Not related to the feature! It was a fix to have green build